chaco/api.py

```diff
@@ -306,6 +384,91 @@
 # Tools
 from .abstract_controller import AbstractController
 
+# Colormaps and color palettes
+from .default_colormaps import (
+    center,
+    color_map_dict,
+    color_map_functions,
+    color_map_name_dict,
+    reverse,
+    jet,
```

Comment:
i'm sorry for not being clear. IIRC, `jet` isnt the first color function defined in the `default_colormaps` module. Can you check the order of functions vs the list at the end of the module in `default_colormaps` - and fix it if necessary - and then update this import statement to reflect the same changes.

Reply:
Yep will do

Reply:
Even further, there are color maps that don't make it into the list / into __all__
Should I add them?  It is unclear if that was intentional or accidental.  For example `binary`

Reply:
definitely add them.

Reply:
Sounds good, thats what I ended up doing in the last commit, and it looks like it was just `binary`